beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -852,6 +919,12 @@ impl<T: BeaconChainTypes> BeaconProcessor<T> {
                         // required to verify some attestations.
                         } else if let Some(item) = gossip_block_queue.pop() {
                             self.spawn_worker(item, toolbox);
+                        // Check sync committee messages before attestations as they are good for only
```

Comment:
The reward for sync committee is lower than attestation though, so not sure of the preference order here.

Reply:
I reckon lets check attestations first, as they're more important: they influence fork choice and can help us follow the head. The reward being higher is another good reason to prioritise them.